packages/amplify-e2e-tests/src/__tests__/parameter-store-2.test.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  addAuthWithEmailVerificationAndUserPoolGroupTriggers,
+  addFunction,
+  amplifyPull,
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  expectParametersOptionalValue,
+  generateRandomShortId,
+  getAppId,
+  getProjectMeta,
+  getTeamProviderInfo,
+  gitCleanFdx,
+  gitCommitAll,
+  gitInit,
+  initJSProjectWithProfile,
+} from '@aws-amplify/amplify-e2e-core';
+
+describe('parameters in Parameter Store', () => {
+  let projRoot: string;
+  const envName = 'enva';
+
+  beforeEach(async () => {
+    projRoot = await createNewProjectDir('multi-env');
```

Comment:
attempt to shorten this due to 
![image](https://user-images.githubusercontent.com/5849952/222593807-b52ef87d-e5e0-41ec-8812-f2a918c92038.png)